packages/x-date-pickers/src/DesktopDatePicker/tests/field.DesktopDatePicker.test.tsx

```diff
@@ -138,18 +138,34 @@ describe('<DesktopDatePicker /> - Field', () => {
       Component: DesktopDatePicker,
     });
 
-    it('should allow to override the placeholder (v6 only)', () => {
-      renderWithProps({
-        enableAccessibleFieldDOMStructure: false,
-        slotProps: {
-          textField: {
-            placeholder: 'Custom placeholder',
+    describe('Placeholder override (v6 only)', () => {
```

Comment:
```suggestion
    describe('placeholder override (v6 only)', () => {
```

Super small nitpick :laughing: